packages/actions/test/index.test.ts

```diff
@@ -63,7 +63,65 @@ describe("Sample e2e", () => {
         assert.isRejected(checkParticipantForCeremony({ ceremonyId: selectedCeremony.id }))
     })
 
-    afterEach(async () => {
+    it('should fail to create a sample ceremony without being a coordinator', async () => {
+        const ceremonyData = fakeCeremoniesData.fakeCeremonyOpenedDynamic
+        const circuitData = fakeCircuitsData.fakeCircuitSmallNoContributors
+        const ceremonyPrefix = ceremonyData.data.title.replace(/[`\s~!@#$%^&*()|+\-=?;:'",.<>\{\}\[\]\\\/]/gi, "-").toLowerCase()
+        
+        // 1 get the bucket
+        const bucket = getBucketName(ceremonyPrefix)
+
+        assert.isRejected(createS3Bucket(userFunctions, bucket))
```

Comment:
This should be enough to justify that if you're not a coordinator, you cannot create a ceremony (and a bucket too, obv).